vm/src/dictdatatype.rs

```diff
@@ -9,20 +9,29 @@ use num_traits::ToPrimitive;
 /// And: http://code.activestate.com/recipes/578375/
 use std::collections::HashMap;
 
-#[derive(Default)]
-pub struct Dict {
+pub struct Dict<T = PyObjectRef> {
```

Comment:
When would you require a different value type? What's the reason to use type parameters?

Reply:
The idea is that the backing type for set will be:

```
type SetContentType = Dict<()>
```

Reply:
Ah, okay, I get it.